cli/cmd/vuln_host.go

```diff
@@ -616,60 +549,73 @@ func hostVulnCVEsTable(cves []api.HostVulnCVE) ([][]string, string) {
 	return out, ""
 }
 
-func hostVulnCVEsTableForSeverity(cves []api.HostVulnCVE, severity string) ([][]string, int, int) {
+func hostVulnCVEsTable(cves []api.HostVulnCVE) [][]string {
+	var out [][]string
+
+	for _, cve := range cves {
+		for _, pkg := range cve.Packages {
+			out = append(out, []string{
+				cve.ID,
+				pkg.Severity,
+				pkg.CvssScore,
+				pkg.Name,
+				pkg.Version,
+				pkg.FixedVersion,
+				pkg.Namespace,
+				pkg.HostCount,
+				pkg.PackageStatus,
+				pkg.Status,
+			})
+		}
+	}
+
+	//order by total number of host
```

Comment:
nit: comments should always have space in between
```suggestion
	// order by the total number of host
```